### PR TITLE
Enable Caffe2 ROCm test on centos

### PR DESCRIPTION
**Pull request by bddppq**
@xw285cornell @petrex @ashishfarmer @rohithkrn

### CODE REVIEW

**Comment by bddppq** [comment]
https://ci.pytorch.org/jenkins/job/caffe2-builds/job/py2-devtoolset7-rocmrpm-centos7.5-trigger-test/23/

File: .jenkins/caffe2/test.sh [hunk 1]
```diff
@@ -113,6 +113,13 @@ if [[ $BUILD_ENVIRONMENT == *-rocm* ]]; then
   rocm_ignore_test+=("--ignore $CAFFE2_PYPATH/python/operator_test/piecewise_linear_transform_test.py")
   rocm_ignore_test+=("--ignore $CAFFE2_PYPATH/python/operator_test/softmax_ops_test.py")
   rocm_ignore_test+=("--ignore $CAFFE2_PYPATH/python/operator_test/unique_ops_test.py")
+
+  if [[ $BUILD_ENVIRONMENT == *-centos* ]]; then
+    # TestTransform.test_apply_transform_if_faster is crashing, but
```
Comment (bddppq) [inline .jenkins/caffe2/test.sh]: Verifying whether this is a general failure on centos in #14091

**Comment by bddppq** [comment]
Once this is merged I will enable the jenkins job separately.

**Comment by petrex** [comment]
Thanks, @bddppq. We want CI /Tests; the more the better.

**Review by yinghai — APPROVED** [review]

**Comment by bddppq** [comment]
Confirmed in #14128 that the failure is unrelated to rocm. Fixed the issue and re-enabled the skipped test there.

**Pull request closed without merging**